### PR TITLE
Remove useless index and addr from RzBinClass

### PR DESCRIPTION
**Pull request by wargio**
**Your checklist for this pull request**
- [ ] I've read the [guidelines for contributing](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md) to this repository
- [ ] I made sure to follow the project's [coding style](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md#code-style)
- [ ] I've documented or updated the documentation of every function and struct this PR changes. If not so I've explained why.
- [ ] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the [rizin book](https://github.com/rizinorg/book) with the relevant information (if needed)

**Detailed description**

Removes some useless info from class.
So `addr` was mostly used to store either a phy or virt addr from where the info was parsed in the bin but the actual value for this is zero to the reverser.

IDA, BN and Ghidra do not have nor provide info about classes.

### CODE REVIEW

File: test/db/cmd/cmd_tc [hunk 1]
```diff
@@ -168,8 +166,6 @@ FILE=bins/mach0/objc-employee-ios14-arm64
 CMDS=ic*
 EXPECT=<<EOF
 fs classes
-"f class.public.Employee @ 0x100007958"
```
Comment (XVilka) [inline test/db/cmd/cmd_tc:171]: Why this disappears?

**Review by XVilka — CHANGES_REQUESTED** [review]

File: test/db/cmd/cmd_tc (hunk 1 above)
Comment (wargio) [reply test/db/cmd/cmd_tc:171]: because before there was a flag for each `addr`, now since we remove it, we don't anymore.

**Comment by ret2libc** [comment]
> Removes some useless info from class.
> So `addr` was mostly used to store either a phy or virt addr from where the info was parsed in the bin but the actual value for this is zero to the reverser.

What was this address like usually?

**Comment by wargio** [comment]
usually for any bin besides mach0 with objc compiled code is either 0 or UT64_MAX.
seems that this address is a virtual address used only for objc classes to create xrefs.
See chat over mm.

**Comment by XVilka** [comment]
As @thestr4ng3r determined it's still necessary for some cases, should be this one closed?

**Pull request closed without merging**